contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViewConstants.java

```diff
@@ -248,6 +248,34 @@
           Arrays.asList(RPC_METHOD),
           CUMULATIVE);
 
+  /**
+   * Cumulative {@link View} for started client RPCs.
+   *
+   * @since 0.12
+   */
+  public static final View RPC_CLIENT_STARTED_COUNT_CUMULATIVE_VIEW =
+      View.create(
+          View.Name.create("grpc.io/client/started_count/cumulative"),
+          "Number of started client RPCs",
+          RPC_CLIENT_STARTED_COUNT,
+          MEAN,
```

Comment:
Why not SUM/COUNT?

Reply:
Because we want to keep both sum and count for cumulative views.

Reply:
Not sure I understand how Sum and Count will be different in this case?

Reply:
We had a discussion with @jcd2 on this long time ago. Initially we decided to use Distribution for all views with histograms and Mean for all others, but I agree using a Count in this case will be more clear. I updated all started/finished count view to use COUNT.